packages/suite/src/components/suite/hocNotification/index.tsx

```diff
@@ -4,7 +4,7 @@ import { SUITE } from '@suite-actions/constants';
 import { NotificationEntry } from '@suite-reducers/notificationReducer';
 import withAction from './components/withAction';
 import withTransaction from './components/withTransaction';
-import withCoinProtocolScheme from './components/withCoinProtocolScheme';
+import { withAoppProtocol, withCoinProtocol } from './components/withCoinProtocolScheme';
```

Comment:
it is not `withCoinProtocolScheme` file anymore. Maybe `withUriProtocolHandler`?